overlord/devicestate/handlers_install.go

```diff
@@ -1010,6 +1012,30 @@ func (m *DeviceManager) doInstallFinish(t *state.Task, _ *tomb.Tomb) error {
 	}
 	defer unmountParts()
 
+	if !sys.Model.Classic() {
+		sysSeed, err := seedOpen(dirs.SnapSeedDir, sys.Label)
+		if err != nil {
+			return fmt.Errorf("cannot open seed: %w", err)
+		}
+
+		if copier, ok := sysSeed.(seed.Copier); ok {
+			db := assertstate.DB(st)
+			err = copier.LoadAssertions(db, func(b *asserts.Batch) error {
```

Comment:
we don't want to get these assertions into the main db I think, it should be fine for this seed to use a temporary db

Reply:
Seed interface already specifies that passing nil as db will result in using a temporary DB, simply `copier.LoadAssertions(nil, nil)` should do it AFAIU